formidable/apps.py

```diff
@@ -1,10 +0,0 @@
-from django.apps import AppConfig
-
-
-class FormidableConfig(AppConfig):
-
-    def ready(self):
-        # Load every interpreter type
-        from formidable.forms.validations.functions import Function  # noqa
-        # Load every interpreter type
-        from formidable.forms.validations.interpreter import *  # noqa
```

Comment:
Like my commit does not say, this AppRegistry is not used. Moreover the Function and interperter modules has been removed couple months ago.

May be a side PR could be more regular, but this is not used and worst, the import in it are not valid.